src/WireFormat.h

```diff
@@ -148,6 +154,17 @@ enum ControlOp {
 };
 
 /**
+ * Used in linearizable RPCs to check whether or not the RPC can be processed.
+ */
+struct ClientLease {
+    uint64_t leaseId;           /// A cluster unique id for a specific lease.
```

Comment:
Any special cases of this value you'd like to tell us about? If you can't think of any, then let's discuss.